README.md

```diff
@@ -51,6 +127,24 @@ pytest
 
 > **Note**
 >
-> You will need `pxr` pre-installed into your python environment in order
-> to run these tests. If you have installed USD in the standard manner,
-> it is likely your `PYTHONPATH` will already be extended appropriately.
+> Refer to the [Running](#running) section for the environmental
+> prerequisites to run these tests.
+
+## Project State
+
+This resolver is currently in prototype stage, it has several key
+limitations.
+
+- Supports only `LocatableContent` traits. Notably, file extensions and
+  file modification times will defer to file-based handling if possible,
+  and return sensible defaults if not. Many potentially expected
+  structures (ie, `AssetInfo`), will be empty.
+
+- `LocatableContent` traits must be provided as absolute paths by the
+  asset manager.
```

Comment:
[fix] Can we clarify that that is the definition of the LocatableContentTrait, rather than a limitation of the current implementation? Maybe its worth splitting that stuff into a section on "How entites are resolved", where we can say that currently we just grab the locatable content trait. Then we can elaborate on the other thing well do in the future (other traits, getRelatedRefrences, etc.)

Then the limitations are more functionally focused?